src/ResourceManager/Profile/Commands.Profile/Context/ImportAzureRMContext.cs

```diff
@@ -0,0 +1,100 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication.Models;
+using Microsoft.Azure.Commands.Profile.Models;
+using Microsoft.Azure.Commands.Profile.Properties;
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using Microsoft.WindowsAzure.Commands.Common;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Profile
+{
+    /// <summary>
+    /// Selects Microsoft Azure profile.
+    /// </summary>
+    [Cmdlet(VerbsData.Import, "AzureRmContext", SupportsShouldProcess = true), OutputType(typeof(PSAzureProfile))]
+    [Alias("Select-AzureRmProfile")]
+    [Obsolete("Select-AzureRmProfile will be renamed to Import-AzureRmContext in the next release.")]
```

Comment:
That was the change I was going to make, but looking at the verb defintions, Import is more approrpiate.  Since we are making a reakign change, in the next release, I thought we should use the most appropriate verb and noun